lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/service/KmsRsaDecrypter.java

```diff
@@ -48,7 +48,8 @@ public byte[] decrypt(
             Base64URL encryptedKey,
             Base64URL iv,
             Base64URL cipherText,
-            Base64URL authTag)
+            Base64URL authTag,
+            byte[] aad)
```

Comment:
This was a breaking change - the `aad` is a new parameter we need to include. We're not using it though so just need to update the signature to ensure we're implementing the interface.